client/src/components/TransactionItem/TransactionItem.tsx

```diff
@@ -60,9 +60,8 @@ const TransactionItem: React.FC<Transaction> = ({
         <ItemName>{title}</ItemName>
         <ItemAmount
           textColor={pageType === "income" ? "#25a244" : "#ff595e"}
-        >{`${currencyFormatter.format(amount)}`}</ItemAmount>
+        >{`${currencyFormatter.format(Math.abs(amount))}`}</ItemAmount>
```

Comment:
So this is formatting so that UI will show the amount in a currency format. The Math.abs() is being used so that we don't show '-$500' - instead, it will show '$500'

I figured showing the negative sign is a bit redundant since the color already differentiates whether its an income or expense on the UI

Reply:
Sweet! Good call! I should have known this, but it was late.